resources/lib/template.py

```diff
@@ -344,7 +344,10 @@ def getProperties( self, elem, items ):
                     continue
                     
                 # We've matched a property :)
-                properties[ name ] = property.text
+                if property.text is not None and property.text != "":
```

Comment:
I prefer the more pythonic "if not property.text:" in cases like that. probably matter of taste. :)